src/NzbDrone.Core/Indexers/IndexerBase.cs

```diff
@@ -23,6 +23,7 @@ public abstract class IndexerBase<TSettings> : IIndexer
         public abstract string Name { get; }
         public abstract DownloadProtocol Protocol { get; }
         public int Priority { get; set; }
+        public int MaximumSingleEpisodeAge { get; set; }
```

Comment:
Since this is more about season packs than single episodes I think this should be renamed to be more inline with that, I'm struggling a bit for a suggestion there though.

Reply:
Yeah, I had considered that too, but couldn't come up with a good wording to represent it. How about just adding "SeasonSearch" in front, like so:
"SeasonSearchMaximumSingleEpisodeAge"
?

Reply:
Yeah, I like that.